### PR TITLE
Fix 9x rotary encoder source 

### LINKED ISSUE #7433: 9XRPRO 2.3 switch mappings
When I try to setup switches they seem to be named incorrectly. This is happening in companion as well. Ex:  3POS says Ail slider, RUD says THR, ELE says RUD.

### LINKED ISSUE #7563: Logical switch always True if variable (V1) as Timer1  (V2) as  (any setting time)
**Describe the bug**
Timer1 countdown not triggering available: (9xrpro)

     Function   V1            V2
L01  a<x           Timer1     0:04:10 [h:mm:ss]

**To Reproduce**
1. setting on remote ctrl (9xrpro)
2. set model menu countdown as Timer1 04:00
3. set L01 a<x Timer1 00:00:10 (10 seconds before end)
4. L01 always on any time.

**Screenshots**
-

**General information**
 - OpenTX version (2.3.7)
 - Hardware (9Xr Pro)
 - OpenTX Companion (e.g. 2.3.7 on W10)

Thank you.

### LINKED ISSUE #7564: Logical switch batt. variable is clock format
**Describe the bug**
Logical sw. Batt. is time format.  9.8V unavailable,  9.59 continued 10:00, not 9.6

**To Reproduce**
1. power on 9xr pro radio
2. set any logical sw v1=batt
3. try set v2 9.8V   (v2 not number, v2 is time format.)

**General information**
 - OpenTX version (2.3.7)
 - Hardware (9Xr Pro)
 - OpenTX Companion (2.3.7 on w10)

thank you

### PR DESCRIPTION
**Pull request by dgatf**
This PR fixes #7563, #7564, #7433, #7199

The rotary encoder was removed from the model sources list in 8aa596e. The entry in the multi string and in companion sources needs to be removed as it was causing a string shift

I've added #7564 to 2.3.8 milestone. When affecting several issues, should I add the other issues to the milestone? or the PR instead?

### CODE REVIEW

**Comment by dgatf** [comment]
Is there a way to restart travis without commiting when it fails for other reason than the code?

**Comment by 3djc** [comment]
You can always ask me, doing it

**Comment by dgatf** [comment]
Thanks

File: companion/src/firmwares/opentx/opentxeeprom.cpp [hunk 1]
```diff
@@ -128,7 +128,7 @@ inline int SWITCHES_CONFIG_SIZE(Board::Type board, int version)
   return 16;
 }
 
-#define MAX_ROTARY_ENCODERS(board)            (IS_SKY9X(board) ? 1 : 0)
```
Comment (bsongis) [inline companion/src/firmwares/opentx/opentxeeprom.cpp:131]: I wonder if the whole thing can be removed, only the gruvin9x had this at the time, it's old

**Comment by bsongis** [comment]
I think you should be able to restart travis now you are in the commiters list

**Comment by bsongis** [comment]
This PR could do far further:
1) remove SWITCH_TYPE_ROTARY_ENCODER
2) remove SOURCE_TYPE_ROTARY_ENCODER
3) remove RotaryEncoders (capability)
4) remove re_label and re_CB
5) remove populateRotEncCB

**Comment by bsongis** [comment]
I will push another PR for this list, it will need to be checked carefully, for 2.3.9

**Comment by dgatf** [comment]
Yes, I agree with that. I found strange to have the encoder as a source (though it was almost removed) and as a switch

**Pull request merged**